### PR TITLE
DT-430 Migrated LightsOff to Core22/Gnome42

### PR DESCRIPTION
**Pull request by sergio-costas**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please check only the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by sergio-costas** [comment]
@hellsworth Removed the 'craftctl set version' statement.

File: snapcraft.yaml [hunk 1]
```diff
@@ -37,33 +37,20 @@ apps:
...
     override-build: |
-      sed -i.bak -e 's|Icon=org.gnome.LightsOff$|Icon=${SNAP}/meta/gui/org.gnome.LightsOff.svg|g' $SNAPCRAFT_PART_SRC/data/org.gnome.LightsOff.desktop.in
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp ../src/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp $SNAPCRAFT_PART_BUILD/data/org.gnome.LightsOff.desktop $SNAPCRAFT_PART_INSTALL/meta/gui/
+      sed -i.bak -e 's|Icon=org.gnome.LightsOff$|Icon=${SNAP}/meta/gui/org.gnome.LightsOff.svg|g' $CRAFT_PART_SRC/data/org.gnome.LightsOff.desktop.in
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $CRAFT_PART_INSTALL/meta/gui/
```
Comment (hellsworth) [inline snapcraft.yaml]: Should we use $CRAFT_PART_SRC here, just to be consistent, like this:
```
cp $CRAFT_PART_SRC/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $CRAFT_PART_INSTALL/meta/gui/
```

**Review by hellsworth — CHANGES_REQUESTED** [review]
Potentially one change for consistency.

File: snapcraft.yaml (hunk 1 above)
Comment (sergio-costas) [reply snapcraft.yaml]: Done.

**Comment by hellsworth** [comment]
Looks good, thanks!

**Pull request merged**